services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/service/impl/OnDemandServiceImpl.java

```diff
@@ -84,31 +84,38 @@ public HostGoalState retrieveGoalState(Goalstateprovisioner.HostRequest.Resource
         logger.log(Level.INFO, "[retrieveGoalState] vni = " + vni +
                 " | sourceIp = " + sourceIp +
                 " | destinationIp = " + destinationIp);
-
+        long start = System.currentTimeMillis();
 //        ResourceMeta resourceMetadata = retrieveResourceMeta(vni, sourceIp);
         VpcResourceMeta vpcResourceMetadata = retrieveResourceMeta(vni);
         if (vpcResourceMetadata == null) {
             logger.log(Level.INFO, "[retrieveGoalState] retrieved vpc resource metadata is null | vni = " + vni);
             return null;
         }
-
+        long end = System.currentTimeMillis();
```

Comment:
We would need an empty line after "}" (Line 93).

Reply:
Added